platform/src/main/resources/patterns/README.md

```diff
@@ -0,0 +1,44 @@
+**Grok**
```

Comment:
Is this the right place to put this documentation? 
We should probably add this to the userManual.src instead, and create a new section in the "loading data" of that document that links to this section. In fact, you should document the new menu in the "loading data" section.

Reply:
@mbudiu-vmw Should I change userManual.md as well?  Looks like there are many line numbers references so need to change the entire file.